### PR TITLE
add alumni page

### LINKED ISSUE #2: [Website] Alumni Page

### PR DESCRIPTION
**Pull request by mduw**
**Updates:**
- Added header/footer 
- Added media queries for devices

---
- New items: 
``students.html`` 
``studentstyle.css``
img folder contains images used by ``students.html``

- ``students.html`` does not include nav bar and footer. 
- student's contact has links to email and Linkedin as of now.

### CODE REVIEW

**Review by isabelgiang — COMMENTED** [review]
I'd remove the `.DS_Store` file or add it to a `.gitignore` so it doesn't clutter up the repository

File: students.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+    <h1>Alumni</h1>
+    <h3>Description for alumni page </h3>
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ac metus elit. Suspendisse pellentesque tellus imperdiet, faucibus tellus at, mattis nisl.
+      Aliquam vestibulum augue vitae libero rhoncus, ac suscipit sem venenatis. Curabitur eu ligula auctor, pellentesque metus nec, venenatis mi. Proin congue nibh sed tellus luctus, 
+      sed mattis mi dictum. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ac metus elit. Suspendisse pellentesque tellus imperdiet, faucibus tellus at, mattis nisl.
+      Aliquam vestibulum augue vitae libero rhoncus, ac suscipit sem venenatis. Curabitur eu ligula auctor, pellentesque metus nec, venenatis mi. Proin congue nibh sed tellus luctus, 
+      sed mattis mi dictum.</p>
+    
+    <div clas="alumni-list">
```
Comment (isabelgiang) [inline students.html]:
```suggestion
    <div class="alumni-list">
```

File: students.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus ac metus elit. Suspendisse pellentesque tellus imperdiet, faucibus tellus at, mattis nisl.
+              Aliquam vestibulum augue vitae libero rhoncus, ac suscipit sem venenatis. Curabitur eu ligula auctor, pellentesque metus nec, venenatis mi. Proin congue nibh sed tellus luctus, 
+              sed mattis mi dictum.
+            </p>
+            <div class="contact-box">
+              <!---
+              More info goes here
+  
+              -->
+              <p id="contact-alignment">Email: <a href="mailto: husky@uw.edu" style="text-decoration: none;">husky@uw.edu</a> <br>
```
Comment (isabelgiang) [inline students.html]: I'd consider moving these `style` attributes out into the `studentstyle.css` file. Makes it easier to manage if all of the styling for the page is in one place.

**Review by isabelgiang — CHANGES_REQUESTED** [review]
Looks fine overall. Just a few small changes requested.

File: students.html (hunk 2 above)
Comment (mduw) [reply students.html]: agree! styles for ``<a>`` are moved to ``studentstyle.css``

File: students.html (hunk 1 above)
Comment (mduw) [reply students.html]: fixed!

Comment (mduw) [reply students.html]: > Suggested change 
>     <div clas="alumni-list">
>     <div class="alumni-list">

fixed!

**Review by redpandarevolt — CHANGES_REQUESTED** [review]
Cut and paste code for <header>....</header>  at the top of <body></body> section.

**Review by redpandarevolt — CHANGES_REQUESTED** [review]
Cut and paste code for <footer>....</footer>  at the bottom  of <body></body> section.

**Review by redpandarevolt — CHANGES_REQUESTED** [review]
1) Add media Query for smart phone 
2) Add media Query for tablet
3)Add media query for laptop
4) Add media query larger screens

https://css-tricks.com/snippets/css/media-queries-for-standard-devices/
We can do some over zoom if you need help.  
See CSS on home page pull request for an example. 
Holla at me if ya' wanna do some examples one zoom.

**Comment by redpandarevolt** [comment]
Screen Shot 2020-07-20 at 11 58 02 AM
Screen Shot 2020-07-20 at 11 58 12 AM
Screen Shot 2020-07-20 at 11 57 45 AM

**Comment by mduw** [comment]
> Cut and paste code for
> 
> .... at the top of section.

footer and header added!

**Comment by mduw** [comment]
> 1. Add media Query for smart phone
> 2. Add media Query for tablet
>    3)Add media query for laptop
> 3. Add media query larger screens
> 
> https://css-tricks.com/snippets/css/media-queries-for-standard-devices/
> We can do some over zoom if you need help.
> See CSS on home page pull request for an example.
> Holla at me if ya' wanna do some examples one zoom.

queries added!

**Comment by mduw** [comment]
Screen Shot 2020-08-03 at 8 25 04 PM
Screen Shot 2020-08-03 at 8 24 55 PM

**Review by redpandarevolt — CHANGES_REQUESTED** [review]
1) Add margin to top paragraph to words are not stuck to edges of page:
 
` margin: <top>px  <right>px   <bottom>px    <left>px; `

2) add some padding into the light purple writing inside of the box on each card so
it doesn't look like its right up against the edges.

` padding: <top>px  <right>px   <bottom>px    <left>px; `

Resolved!

**Review by isabelgiang — APPROVED** [review]

**Comment by mduw** [comment]
this PR is out-of-date. See the #9 for update.

**Pull request closed without merging**